docs/guides/qpu-information.ipynb

```diff
@@ -0,0 +1,805 @@
+    "| |Changing the direction of a CNOT gate | |\n",
+    "\n",
+    "<Tabs>\n",
+    "  <TabItem value=\"Qiskit-backend\" label=\"View version with Qiskit\">\n",
+    "\n",
+    "    <CodeCellPlaceholder tag=\"id-version\" />\n",
+    "\n",
+    "  </TabItem>\n",
+    "  <TabItem value=\"IQP-backend\" label=\"View version on IBM Quantum Platform\">\n",
+    "\n",
+    "    Navigate to the [Compute resources](https://quantum.cloud.ibm.com/computers) page to view a table of all QPUs, then click the name of any QPU to open its detailed information card. Find the QPU version under the Details section.\n",
+    "\n",
+    "  </TabItem>\n",
+    "</Tabs>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "8d070b69-1b6d-4faa-9a4a-d12aba47d0b7",
+   "metadata": {
+    "tags": [
+     "id-version"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Name: ibm_fez\n",
+      "Version: 2\n",
+      "No. of qubits: 156\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "backend = service.backend(\"ibm_fez\")\n",
+    "\n",
+    "print(\n",
+    "    f\"Name: {backend.name}\\n\"\n",
+    "    f\"Version: {backend.version}\\n\"\n",
+    "    f\"No. of qubits: {backend.num_qubits}\\n\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a055938-0a1d-46ef-ae9c-60a7960d6d73",
+   "metadata": {},
+   "source": [
+    "<span id=\"native-gates-ops\"></span>\n",
+    "### Native gates and operations\n",
+    "\n",
+    "Each [processor family](/docs/guides/processor-types) has a native gate set. By default, the QPUs in each family only support running the gates and operations in the native gate set. Thus, every gate in the circuit must be translated (by the transpiler) to the elements of this set.\n",
+    "\n",
+    "You can view the native gates and supported instructions, including non-gate-type instructions, for a QPU with Qiskit. The IBM Quantum Platform Compute resources page lists the basis gates for a QPU.\n",
+    "\n",
+    "\n",
+    "  <Tabs>\n",
+    "  <TabItem value=\"Qiskit-native\" label=\"Find native gates with Qiskit\">\n",
+    "\n",
+    "    <span id=\"native-gates-with-qiskit\"></span>\n",
+    "    ```python\n",
+    "\n",
+    "    from qiskit_ibm_runtime import QiskitRuntimeService\n",
+    "\n",
+    "    service = QiskitRuntimeService()\n",
+    "\n",
+    "    for backend in service.backends():\n",
+    "        config = backend.configuration()\n",
+    "        if \"simulator\" in config.backend_name:\n",
+    "            continue\n",
+    "        print(f\"Backend: {config.backend_name}\")\n",
+    "        print(f\"    Processor type: {config.processor_type}\")\n",
+    "        print(f\"    Supported instructions:\")\n",
+    "        for instruction in config.supported_instructions:\n",
+    "            print(f\"        {instruction}\")\n",
+    "        print()\n",
+    "    ```\n",
```

Comment:
This code block should be simplified to

```python
from qiskit_ibm_runtime import QiskitRuntimeService

service = QiskitRuntimeService()
backend = service.backend("ibm_fez")

print(f"Backend: {backend.name}")
print(f"    Processor type: {backend.processor_type}")
print(f"    Supported instructions: {backend.supported_instructions}")
```

because (1) the old code loops through all backends, which is very slow and (2) you can extract backend information directly; no need to get its configuration.